submissions/tre-jeremy/scripts/views/articleView.js

```diff
@@ -0,0 +1,159 @@
+(function(module) {
+
+  var articleView = {};
+
+  var render = function(article) {
+    var template = Handlebars.compile($('#article-template').text());
+
+    article.daysAgo = parseInt((new Date() - new Date(article.publishedOn))/60/60/24/1000);
+    article.publishStatus = article.publishedOn ? 'published ' + article.daysAgo + ' days ago' : '(draft)';
+    article.body = marked(article.body);
+
+    return template(article);
+  };
+
+  // COMMENTs: What does this method do?  What is it's execution path?
+// fills the filter templates from the DB, discarding duplicates. Fired by articleView.index
```

Comment:
How do each of the methods for populating the filters work? What are the pros and cons of each?